client/src/constants/index.ts

```diff
@@ -1 +1,22 @@
+import { InputValidator } from 'types';
+
+import { capitalize } from 'utils/helpers';
+
 export const BASE_URL = 'http://localhost:3000';
+
+export const VALIDATOR: Record<string, (value: string, param: number | boolean) => boolean> = {
```

Comment:
`string` in first argument of `Record` is too common, please replace with more specific type

Reply:
This one again will involve using keyword `as`, as I can't think of any other way. I am thinking of defining a type `InputValidatorKey = keyof typeof InputValidator` and using it like so: `Record<InputValidatorKey, ...>`. Doing so would result in the line `const valid = VALIDATOR[validator[0]](value, validator[1]);` throwing the same error "Element implicitly has an 'any' type because expression of type 'string' can't be used to index type..." in the file `inputValidator.ts`, and the solution that I know would be to set `validator[0] as InputValidatorKey`, because a couple lines above I am again using `Object.entries()` like so: `const validators = Object.entries(validation);`.